src/app/shared/components/headerView.tsx

```diff
@@ -0,0 +1,55 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
+import * as React from 'react';
+import { TooltipHost, DirectionalHint } from 'office-ui-fabric-react/lib/Tooltip';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { IconButton } from 'office-ui-fabric-react/lib/Button';
+import { getId, Link } from 'office-ui-fabric-react';
+import { useLocalizationContext } from '../contexts/localizationContext';
+
+export interface HeaderViewProps {
+    headerText: string;
+    tooltip?: string;
+    link?: string;
+}
+
+export const HeaderView: React.FC<HeaderViewProps> = props => {
+    const hostId = getId('tooltip');
```

Comment:
why is this needed?

Reply:
for accessibility to asscociate the tooltip to the text